map2loop/project.py

```diff
@@ -566,6 +590,7 @@ def save_into_projectfile(self):
                 depth=[self.map_data.bounding_box["top"], self.map_data.bounding_box["base"]],
                 spacing=[1000, 1000, 500],
                 preference="utm",
+                epsg=self.map_data.get_working_projection(),
```

Comment:
As epsg is in the extents here, the version of loopprojectfile in dependencies needs to be updated to >=0.1.2

Reply:
it's now in requirements -  0.1.3 --> https://github.com/Loop3D/map2loop/pull/60/commits/cbb54899368ae5950b29a101bfca648b900d6527